src/Api/FhirFileStorageInfo.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright 2021 MONAI Consortium
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Ardalis.GuardClauses;
+using Monai.Deploy.InformaticsGateway.Api.Rest;
+using System.IO.Abstractions;
+
+namespace Monai.Deploy.InformaticsGateway.Api
+{
+    /// <summary>
+    /// Provides basic information for a DICOM instance and storage hierarchy/path.
```

Comment:
Comments are for the DICOM file storage info.

Reply:
Updated; thanks.